tracer/src/Datadog.Tracer.Native/debugger_probes_instrumentation_requester.cpp

```diff
@@ -270,6 +271,12 @@ void DebuggerProbesInstrumentationRequester::AddMethodProbes(
         {
             const DebuggerMethodProbeDefinition& current = methodProbes[i];
 
+            if (IsProbeIdExist(current.probeId))
+            {
+                Logger::Info("[AddMethodProbes] Method Probe Id: ", current.probeId, " is already processed.");
```

Comment:
Should this be `DEBUG`? Don't we risk flooding the log file here?

Reply:
Done in f2e39f19694d2e9a44c777ae15e0fa38a3d73526.